window-copy.c

```diff
@@ -3097,19 +3116,60 @@ window_copy_search(struct window_mode_entry *wme, int direction, int regex,
 	wrapflag = options_get_number(wp->window->options, "wrap-search");
 	cis = window_copy_is_lowercase(str);
 
-	if (direction)
+	if (direction) {
+		keys = options_get_number(wp->window->options, "mode-keys");
+		/*
+		 * Behave according to mode-keys. If it is emacs, search forward leaves the
```

Comment:
What is the standard tab width and line width in this codebase?

Reply:
I think https://man.openbsd.org/style is what we follow.

Reply:
Thanks, formatting adjusted accordingly.